src/algorithms.jl

```diff
@@ -4406,8 +4406,7 @@ Scientific Computing, 18 (1), pp. 1-22.
 différentiels-algébriques. MSc mathematics thesis, Faculty of Science,
 University of Geneva, Switzerland.
 =#
-_unwrap_val(::Val{B}) where {B} = B
-_unwrap_val(B) = B
+import SciMLBase._unwrap_val
```

Comment:
Could this be moved to the other imports in src/OrdinaryDiffEq.jl? And maybe don't use `import` since we don't want to extend it?

Reply:
Yeah, I would prefer that as well.

Reply:
done.